arches/app/views/base.py

```diff
@@ -52,7 +52,9 @@ def get_context_data(self, **kwargs):
             if self.request.user.has_perm("view_plugin", plugin):
                 context["plugins"].append(plugin)
 
-        createable = get_createable_resource_types(self.request.user)
+        createable = list(
+            models.GraphModel.objects.filter(pk__in=list(get_createable_resource_types(self.request.user))).all()
```

Comment:
I get the following error in the report view when I reach this line:
`django.core.exceptions.ValidationError: ['“<model name>” is not a valid UUID.']`

The `get_createable_resource_types` method already returns a list of graphs that a user has access to. 
```
(Pdb) get_createable_resource_types(self.request.user)                            
[<GraphModel: Report>, <GraphModel: Test Concept Load>, <GraphModel: Creature>, <GraphModel: Resource>, <GraphModel: Test>]
```
If I revert this line back to ```createable = get_createable_resource_types(self.request.user)``` the report view loads as expected.

Reply:
It looks like this was a mistake in the rebasing - Thanks @chiatt ! In our fork, I had changed it to list IDs - the main advantage is:

* most uses of `get_createable_resource_types` or `get_nodegroups_by_perm` do not use the full graph object, only a count or IDs
* as they are used in permissions, they get hit every request and we saw a decent speed-up if we could skip a full Graph/NodeGroup load

By making these return strings, the permissions framework can pull back less from the DB in (e.g.) API requests or, if it can cache (as our Casbin implementation) then it can fulfill these without needing to hit the DB at all.

If that logic makes sense then it might be tidier than what I have here, if I move this Django model call into a `get_createable_resource_models` wrapper, and pull in the `get_createable_resource_types` (to return `list[str]`) change from our fork?

Reply:
Thanks Phil - Yeah, that sounds like a really nice improvement.

Reply:
Added in.